robots/text.js

```diff
@@ -72,9 +72,11 @@ function limitMaximumSentences(content) {
 }
 
 async function fetchKeywordsOfAllSentences(content) {
-  for (const sentence of content.sentences) {
-    sentence.keywords = await fetchWatsonAndReturnKeywords(sentence.text)
-  }
+  await Promise.all(
+    content.sentences.map(
+      async sentence => sentence.keywords = await fetchWatsonAndReturnKeywords(sentence.text)
+    )
+  )
```

Comment:
Prata do PR.
Paraleliza fetch de keywords. O `for..of` faz um await de cada vez... o `Promise.all` com `map()` não.